examples/benchmarks/bubblebench/shared-tree/jest.config.cjs

```diff
@@ -4,19 +4,24 @@
  */
 
 // Get the test port from the global map and set it in env for this test
-const testTools = require("@fluidframework/test-tools");
+let mappedPort = 9000; // Defaul port test-tools would provide if no mapping exists
+// Only import test-tools if POLICY_CHECK is not set.
+// This allows us to run policy checks on the jest config files without having to build the repo.
+if (process.env.POLICY_CHECK === undefined) {
+	const testTools = require("@fluid-private/test-tools");
+	mappedPort = testTools.getTestPort(name);
+}
 const { name } = require("./package.json");
 
-mappedPort = testTools.getTestPort(name);
-process.env["PORT"] = mappedPort;
+process.env.PORT = mappedPort;
 
 module.exports = {
 	preset: "jest-puppeteer",
 	globals: {
-		PATH: `http://localhost:${mappedPort}/`,
+		PATH: `http://localhost:${mappedPort}`,
 	},
 	testMatch: ["**/?(*.)+(spec|test).[t]s"],
-	testPathIgnorePatterns: ["/node_modules/"],
+	testPathIgnorePatterns: ["/node_modules/", "dist"],
```

Comment:
This and the change in line 21 make this jest config consistent with all others. I couldn't see an obvious reason why it should be different, so I think these are fine to keep.